examples/shopping-cart/store/modules/product/actions.js

```diff
@@ -0,0 +1,12 @@
+import shop from '../../../api/shop'
+import * as types from './types'
+
+export const getAll = ({ commit }) => {
+  shop.getProducts(products => {
+    commit(types.RECEIVE_PRODUCTS, { products })
+  })
+}
+
+export const decreaseInventory = ({ commit }, product) => {
```

Comment:
We seem not to need this action since it's not dispatched in any places?

Reply:
That's true, but what is the best option? Commit mutations from another module like I did OR dispatch actions from another module?

In my opinion the second option (dispatch actions) is better because we don't need to import the types from another module: `import * as productTypes from '../product/types'`

Reply:
Sorry, I totally forgot this PR.

I actually does not use the constant type pattern in Vuex. But I'd say it seems for preventing mismatch between dispatched (committed) type and action (mutation) name caused by typo, refactoring etc, it is ok to use them across different modules.